python/mdvtools/project_router.py

```
@@ -1,5 +1,6 @@
 import os
-from flask import Flask, Response, request, jsonify, session, redirect, url_for
+import json
```

Ruff (0.8.2)

`json` imported but unused

Remove unused import: `json`

(F401)